components/_preview.hbs

```diff
@@ -2,7 +2,13 @@
     <head>
         <meta charset="utf-8">
         <title>{{ title }}</title>
-        <link media="all" rel="stylesheet" href="{{ path '/css/styles.css' }}">
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="Alpaca components based on gulp and fractal. Part of alpaca project. Required in Alpaca M2 theme">
+        <meta name="keywords" content="components, gulp, js, scss, fractal, magento2">
```

Comment:
It's not necessary, it's just the preview loaded as iframe.

Reply:
ok, removed